spec.html

```diff
@@ -15375,10 +15381,18 @@ <h1>Static Semantics: BoundNames</h1>
       <emu-clause id="sec-destructuring-binding-patterns-static-semantics-containsexpression">
         <h1>Static Semantics: ContainsExpression</h1>
         <emu-see-also-para op="ContainsExpression"></emu-see-also-para>
-        <emu-grammar>ObjectBindingPattern : `{` `}`</emu-grammar>
+        <emu-grammar>
+          ObjectBindingPattern :
+            `{` `}`
+            `{` BindingRestProperty `}`
```

Comment:
Alternatively (and more analogous to _BindingRestElement_), we could return true from _BindingRestProperty_ : `...` _BindingIdentifier_ instead.

Reply:
Why *true*?